src/fcu_io.cpp

```diff
@@ -127,6 +142,28 @@ void fcuIO::handle_small_imu_msg(const mavlink_message_t &msg)
   sensor_msgs::Temperature temp_msg;
   temp_msg.header.stamp = imu_msg.header.stamp;
 
+  static int send_throttle = 0;
+
+  // This is so we can eventually make calibrating the IMU an external service
+  if (!imu_.calibrated)
+  {
+    if(imu_.calibrate(imu))
+    {
+      ROS_INFO("accel parameters found\n xm = %f, ym = %f, zm = %f xb = %f yb = %f, zb = %f",
+               imu_.xm(),imu_.ym(),imu_.zm(),imu_.xb(),imu_.yb(),imu_.zb());
+        // calibration is done, send params to the param server and save them
+        /// DOING IT THIS WAY ENSURES THAT FCU_IO WILL CRASH -- I THINK WE ARE OVERFLOWING A BUFFER SOMEWHERE
+        mavrosflight_->param.set_param_value("ACC_X_TEMP_COMP", 1000.0*imu_.xm());
+        mavrosflight_->param.set_param_value("ACC_Y_TEMP_COMP", 1000.0*imu_.ym());
+        mavrosflight_->param.set_param_value("ACC_Z_TEMP_COMP", 1000.0*imu_.zm());
+        mavrosflight_->param.set_param_value("ACC_X_BIAS", 1000.0*imu_.xb());
+        mavrosflight_->param.set_param_value("ACC_Y_BIAS", 1000.0*imu_.yb());
+        mavrosflight_->param.set_param_value("ACC_Z_BIAS", 1000.0*imu_.zb());
+        mavrosflight_->param.write_params();
```

Comment:
I'm pretty sure this is sorta what both you and I were thinking about how to pass the biases back over to the naze.  Do you know of any reason why this would cause mavros to crash?